lib/pure/options.nim

```diff
@@ -145,13 +145,23 @@ proc `==`*(a, b: Option): bool =
   (a.has and b.has and a.val == b.val) or (not a.has and not b.has)
 
 
-proc `$`*[T]( self: Option[T] ): string =
+proc `$`*[T](self: Option[T]): string =
   ## Returns the contents of this option or `otherwise` if the option is none.
```

Comment:
The comment on this proc seems a little misleading, might as well update it whilst you've got a PR touching the code anyway I reckon.